cicecore/cicedynB/general/ice_forcing.F90

```diff
@@ -2668,247 +2668,6 @@ subroutine JRA55_data
-      ! call interpolate_data (fsw_data, fsw)
-      ! call interpolate_data (flw_data, flw)
-      ! call interpolate_data (fsnow_data, fsnow)
-      fsw(:,:,:) = fsw_data(:,:,1,:)
-      flw(:,:,:) = flw_data(:,:,1,:)
-      fsnow(:,:,:) = fsnow_data(:,:,1,:)
-
-      !$OMP PARALLEL DO PRIVATE(iblk,i,j)
-      do iblk = 1, nblocks
-        ! limit summer Tair values where ice is present
-        do j = 1, ny_block
-          do i = 1, nx_block
-            if (aice(i,j,iblk) > p1) Tair(i,j,iblk) = min(Tair(i,j,iblk), Tffresh+p1)
-          enddo
-        enddo
-
-        do j = 1, ny_block
-          do i = 1, nx_block
-            Qa  (i,j,iblk) = Qa  (i,j,iblk) * hm(i,j,iblk)
-            Tair(i,j,iblk) = Tair(i,j,iblk) * hm(i,j,iblk)
-            uatm(i,j,iblk) = uatm(i,j,iblk) * hm(i,j,iblk)
-            vatm(i,j,iblk) = vatm(i,j,iblk) * hm(i,j,iblk)
-            fsw (i,j,iblk) = fsw (i,j,iblk) * hm(i,j,iblk)
-            flw (i,j,iblk) = flw (i,j,iblk) * hm(i,j,iblk)
-            fsnow(i,j,iblk) = fsnow (i,j,iblk) * hm(i,j,iblk)
-          enddo
-        enddo
-
-      enddo  ! iblk
-      !$OMP END PARALLEL DO
-
-         if (debug_n_d .or. dbug) then
-           if (my_task.eq.master_task) &
-               write (nu_diag,*) subname,'JRA55_bulk_data'
-           vmin = global_minval(fsw,distrb_info,tmask)
-           vmax = global_maxval(fsw,distrb_info,tmask)
-           if (my_task.eq.master_task) &
-               write (nu_diag,*) subname,'fsw',vmin,vmax 
-           vmin = global_minval(flw,distrb_info,tmask)
-           vmax = global_maxval(flw,distrb_info,tmask)
-           if (my_task.eq.master_task) & 
-               write (nu_diag,*) subname,'flw',vmin,vmax
-           vmin =global_minval(fsnow,distrb_info,tmask)
-           vmax =global_maxval(fsnow,distrb_info,tmask)
-           if (my_task.eq.master_task) & 
-               write (nu_diag,*) subname,'fsnow',vmin,vmax
-           vmin = global_minval(Tair,distrb_info,tmask)
-           vmax = global_maxval(Tair,distrb_info,tmask)
-           if (my_task.eq.master_task) & 
-               write (nu_diag,*) subname,'Tair',vmin,vmax
-           vmin = global_minval(uatm,distrb_info,umask)
-           vmax = global_maxval(uatm,distrb_info,umask)
-           if (my_task.eq.master_task) & 
-               write (nu_diag,*) subname,'uatm',vmin,vmax
-           vmin = global_minval(vatm,distrb_info,umask)
-           vmax = global_maxval(vatm,distrb_info,umask)
-           if (my_task.eq.master_task) & 
-               write (nu_diag,*) subname,'vatm',vmin,vmax
-           vmin = global_minval(Qa,distrb_info,tmask)
-           vmax = global_maxval(Qa,distrb_info,tmask)
-           if (my_task.eq.master_task) &
-               write (nu_diag,*) subname,'Qa',vmin,vmax
-
-        endif                   ! dbug
-
-      end subroutine Jra55_data_old
-
 !=======================================================================
 !
 ! AOMIP shortwave forcing
```

Comment:
Just a note, mainly as a reminder:  We are making JRA the default, but the older forcing options are still available.  Those will need to be deprecated via our process (not in this release).